src/cls/rgw/cls_rgw_types.h

```diff
@@ -1046,6 +1046,57 @@ struct cls_rgw_gc_obj_info
 };
 WRITE_CLASS_ENCODER(cls_rgw_gc_obj_info)
 
+extern const char* LC_STATUS[];
+
+typedef enum {
+  lc_uninitial = 0,
+  lc_processing,
+  lc_failed,
+  lc_complete,
+}LC_BUCKET_STATUS;
+
+struct cls_rgw_lc_entry
+{
+  string bucket;
+  int status;
```

Comment:
you can init these members here, instead of in constructor init section, won't need to write it twice; for example: `int status{0};`